packages/jaspr_lints/pubspec.yaml

```diff
@@ -0,0 +1,26 @@
+name: jaspr_lints
+description: A collection of lints and assists for jaspr projects.
+version: 0.1.0
+homepage: https://github.com/schultek/jaspr
+issue_tracker: https://github.com/schultek/jaspr/issues
+documentation: https://docs.page/schultek/jaspr
```

Comment:
Here could be added url exactly into linter page in documentation.

Reply:
Good idea, done